lib/Serialization/Serialization.cpp

```diff
@@ -3743,7 +3743,7 @@ static void writeGroupNames(const comment_block::GroupNamesLayout &GroupNames,
     Writer.write<uint32_t>(N.size());
     BlobStream << N;
   }
-  BlobStream.str();
+  BlobStream.flush();
```

Comment:
The most idiomatic thing to do would probably be to pass `BlobStream.str()` to the `emit` call below, rather than accessing `Blob` directly.